tests/modules/icgc-argo-workflows/song/submit/test.yml

```diff
@@ -1,6 +1,6 @@
 
 - name: "song submit"
-  command: nextflow run ./tests/modules/icgc-argo-workflows/song/submit -entry test_song_submit -c ./tests/config/nextflow.config -profile docker,debug_qa --api_token YOUR_TOKEN
+  command: nextflow run ./tests/modules/icgc-argo-workflows/song/submit -entry test_song_submit -c ./tests/config/nextflow.config -profile docker,rdpc_qa --api_token YOUR_TOKEN
```

Comment:
updated to `rdpc_qa` in `song/submit`. Test run with error.
```
SONG_SERVER_ERROR[schema.violation @ 1724082584904]: [SubmitService::schema.violation] - #/experiment: required key [target_capture_kit] not found,#/experiment: required key [primary_target_regions] not found,#/experiment: required key [capture_target_regions] not found
```

In `test_data.config` `payload = "${params.test_data_base}/9940db0f-c100-496a-80db-0fc100d96ac1.analysis.json"` need to be updated with payload which meet the latest schema requirement.

As this is test is prepared by @edsu7, could you please help update the payload